### PR TITLE
Update to run under python3

### PR DESCRIPTION
**Pull request by kyllikki**

### CODE REVIEW

File: jostedal/stun/agent.py [hunk 1]
```diff
@@ -311,15 +305,15 @@ def __init__(self, data, family, port, address):
     @classmethod
     def decode(cls, data, offset, length):
         family, port = cls.struct.unpack_from(data, offset)
-        packed_ip = buffer(data, offset + cls.struct.size, length - cls.struct.size)
+        packed_ip = memoryview(data)[offset + cls.struct.size : offset + length]
```
Comment (jmb202) [inline jostedal/stun/agent.py:308]: `offset + cls.struct.size : offset + cls.struct.size + length]`?

File: jostedal/stun/attributes.py [hunk 2]
```diff
@@ -69,14 +69,14 @@ def __init__(self, data, err_class, err_number, reason):
...
-        self.reason = str(reason).decode("utf8")
+        self.reason = str(reason)
 
     @classmethod
     def decode(cls, data, offset, length):
         err_class, err_number = cls._struct.unpack_from(data, offset)
         err_class &= 0b111
-        value = buffer(data, offset, length)
-        reason = buffer(value, cls._struct.size)
+        value = memoryview(data)[offset:length]
```
Comment (jmb202) [inline jostedal/stun/attributes.py]: `[offset : offset + length]`?

File: jostedal/stun/agent.py (hunk 1 above)
Comment (kyllikki) [reply jostedal/stun/agent.py:308]: buffer used start:length slice wants start:end so
length to end here is ```length - cls.struct.size``` to ```offset + cls.struct.size + length - cls.struct.size``` which (I think) simplifies to ```offset + length```

Comment (jmb202) [reply jostedal/stun/agent.py:308]: Agreed. Addition is hard.

File: jostedal/stun/attributes.py (hunk 2 above)
Comment (kyllikki) [reply jostedal/stun/attributes.py]: thanks, fixed

**Pull request merged**